Time/timetable.1m.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+
+# xbar meta
+# <xbar.title>Timetable</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Cabin Zhu</xbar.author>
+# <xbar.author.github>cabinz</xbar.author.github>
+# <xbar.desc>Display the current event from a given daily timetable. Make sure to configure the path to your CSV timetable file as the global variable CSV_TIMETAB.</xbar.desc>
+# <xbar.image>https://raw.githubusercontent.com/cabinz/my-xbar-plugins/main/timetable/timetable.png</xbar.image>
+# <xbar.dependencies>python</xbar.dependencies>
+
+import datetime
+import csv
+from dataclasses import dataclass
+from typing import List, Union
+
+# Each line in the format "Begin_Time,End_Time,Event_Name"
+#
+# Example (timetable.csv):
+# 23:30,07:30,sleep
+# 07:30,08:30,breakfast
+# 08:30,10:00,gym
+# ...
+CSV_TIMETAB = "/Applications/SwiftBar/timetable/.timetable.csv"
```

Comment:
This is a good use case for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html).

Reply:
Hi, I've already created a commit to use xbar variabls to make the CSV file path configurable.

Btw, I was the user of [SwiftBar](https://github.com/swiftbar/SwiftBar) and I initially developed this plug-in app for it. It's your change requests made me realize this repo is originally for another earlier open source project called xbar 😂. May I know is there any essential differences / relationship between these two projects?

Moreover, is there any way in xbar to redirect the local plugin folder from the default path `/Users/xxx/Library/Application Support/xbar/plugins` to a customized folder? (e.g. `.../xbar-plugins/Enabled`) I didn't find an obvious entry for this in xbar. Thanks!